qa/common/shm_util.py

```diff
@@ -448,7 +440,7 @@ def __exit__(self, type, value, traceback):
                 if current_shm_size != prev_shm_size:
                     shm_leak_detected = True
                     print(
-                        f"Shared memory leak detected: {current_shm_size} (current) != {prev_shm_size} (prev)."
+                        f"[{self._debug_str}] Shared memory leak detected: {current_shm_size} (current) != {prev_shm_size} (prev)."
```

Comment:
This is just to make it more immediately clear what subtest is leaking in job logs.